client/app/pages/queries/components/QueryPageHeader.less

```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-wrap: wrap;
   align-items: stretch;
-  margin: -5px 0;
+  margin-bottom: -5px;
```

Comment:
This change keeps a margin on the top of the Query Page (to be more consistent with other pages).